examples/webpack.config.local.js

```diff
@@ -54,7 +58,7 @@ const LOCAL_DEVELOPMENT_CONFIG = {
 
   resolve: {
     // Imports the library from its src directory in this repo
-    alias: ALIASES
+    alias: Object.assign({}, ALIASES, LUMA_LOCAL_ALIASES)
```

Comment:
This should not be done by default, it should depend on an env variable, like deck and math.